content/bash-introduction/walkthrough/ls/index.md

```diff
@@ -72,7 +72,7 @@ Try listing the contents of the remaining directories inside of `/home/student/`
 ## `ls` Options
 
 `ls` has many options for you to choose from, however the two most common options are:
-- `--long`: show the permissions, size and date last modified among other meta-data
+- `-l`: show the permissions, size, and date last modified among other meta-data (this is often referred to as long listing format)
```

Comment:
my bad, I didn't realize the `--long` wasn't the long hand form of the `-l` option. Good **correction** fix!